modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase

```diff
@@ -38,13 +38,73 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6024. AddASKUWithSubscriptions"
-	@ignore = "true"
 	@priority = "3"
 	test AddASKUWithSubscriptions {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6024. AddASKUWithSubscriptions
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		Refresh();
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+		CommerceSKUs.addSKU(
+			noOption = "true",
+			optionValuesList = "BLACK:10.0");
+
+		Click(
+			key_value = "BLACKSKU",
+			locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+		MenuItem.click(menuItem = "Edit");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
 
+		Click(locator1 = "CommerceEntry#CLICK_SUBSCRIPTIONS");
+
+		Click(locator1 = "CommerceEntry#CLICK_OVERRIDE");
+
+		Button.clickSave();
+
+		AssertVisible(locator1 = "Message#SUCCESS");
+
+		AssertElementPresent.assertVisible(locator1 = "CommerceEntry#PAYMENT_SUBSCRIPTIONS");
+
+		AssertElementPresent.assertVisible(locator1 = "CommerceEntry#DELIVERY_SUBSCRIPTIONS");
+
+		Pause(locator1 = "3000");
+
+		Click(locator1 = "CommerceEntry#CLICK_OVERRIDE");
+
+		Button.clickSave();
+
+		AssertVisible(locator1 = "Message#SUCCESS");
+
+		AssertNotVisible(locator1 = "CommerceEntry#PAYMENT_SUBSCRIPTIONS");
+
+		AssertNotVisible(locator1 = "CommerceEntry#DELIVERY_SUBSCRIPTIONS");
```

Comment:
Please create a single path with a variable so that it can be used for both cases